### DISCUSSION TITLE
(Other) Permintaan buat script show page all untuk website berita indonesia

### DISCUSSION THREAD

**Discussion by 3xploiton3** [realodix/AdBlockID#727] [General]
Maaf sepertinya melenceng dari tema,

Bang apakah klo bisa tolong buatkan script untuk menampilkan isi berita dalam 1 halaman

Contoh script seperti ini
https://greasyfork.org/en/scripts/389536-show-all-detik-com/code

Banyak website yang pakai
Gridoto.com
Tribunnews.com
Kompas.com
Cnbcindonesia.com
Tempo.co
Dan masih banyak lain,

**Comment by realodix** [comment]
Waduh maaf Bro, Saya tidak familiar dengan Userscripts.

Bro bisa contact @yovir, beliau author dari [Show All detik.com](https://greasyfork.org/en/scripts/389536-show-all-detik-com) dan [Show All Kompas.com](https://greasyfork.org/en/scripts/373617-show-all-kompas-com) di https://greasyfork.org.
